server/src/types/common.rs

```diff
@@ -1,44 +1,37 @@
-use spacetimedb::{Identity, ReducerContext, SpacetimeType, Table};
-
-use super::stellarobjects::player_controlled_stellar_object;
+use spacetimedb::ReducerContext;
+use spacetimedsl::{dsl, Wrapper};
 
+use super::{stellarobjects::{GetPlayerControlledStellarObjectRowOptionByIdentity, StellarObjectId}};
 
 
+#[dsl(plural_name = global_configurations)]
 #[spacetimedb::table(name = global_config)]
```

Comment:
If the plural name is `global_configurations` you should use `global_configuration` as singular name.